src/ui/Button/Button.stories.tsx

```diff
@@ -1,38 +1,51 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { withDesign } from 'storybook-addon-designs';
 
-import { Box } from '../Box/Box';
+import { Flex, Text, Box, TextArea, HR } from 'ui';
 
 import { Button, ButtonStory } from './Button';
 
 export default {
-  title: 'Design System/Components/Button',
   component: Button,
-  decorators: [withDesign],
+  decorators: [
+    withDesign,
+    (Story: any) => (
+      <>
+        <Story />
+        <HR />
+        <Text variant="label">Component in Context</Text>
+        <Box css={{ mt: '$md', p: '$md', background: '$surface' }}>
+          <TextArea css={{ width: '100%', background: 'white' }} />
+          <Flex css={{ justifyContent: 'flex-end', mt: '$md' }}>
+            <Story />
+          </Flex>
+        </Box>
+      </>
```

Comment:
@karelianpie thanks for the review. It feels like some of these components just sitting by themselves are hard to understand or get a good sense for, and they need more context to see them more fully, and to make sure change are properly fixed. Can you say a bit more on your thoughts for this?

Like if this was a form context or a modal context rather than just a textarea, would that make more sense to you? Or in general the additional context feels like an anti-pattern?

Reply:
I went ahead and merged this - but would love to hear your best practices and make changes later on to use Storybook in the best way.

Reply:
All the companies I've worked for had the components in isolation and then a more complex story with its usage in some context. The preferred approach if a component is likely to be used together in some context is writing a story for multiple components, see here https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components

For this case, we could have a story just for the button and then a "Context" or "Usage Example" story for example where the component is used within an example context. Wdyt?

Reply:
Ok, that sounds good.

So have the individual components just be simple and by themselves, and then have a separate set of Stories with examples that import these stories into broader contexts? @karelianpie

Reply:
Yes, and the broader context stories would be [stories for multiple components](https://storybook.js.org/docs/react/writing-stories/stories-for-multiple-components) so that we can still tweak the main component.